Doc/library/stdtypes.rst

```diff
@@ -2404,8 +2404,26 @@ data and are closely related to string objects in a variety of other ways.
       >>> b'\xf0\xf1\xf2'.hex()
       'f0f1f2'
 
+      If you want to make the hex string easier to read, you can specify a
+      single character separator *sep* parameter to include in the output.
+      By default between each byte.  A second optional *bytes_per_sep*
+      parameter controls the spacing.  Positive values calculate the
+      separator position from the right, negative values from the left.
+
+      >>> value = b'\xf0\xf1\xf2'
```

Comment:
```suggestion
     >>> value = b'\xf0\xf1\xf2'
```

Reply:
Sorry, looking at the above examples it seems the other statements have to be shifted right by one space so that they are grouped under hex method like the rest. Please ignore my suggestion.